compiler/next/include/chpl/uast/ASTClassesList.h

```diff
@@ -136,9 +136,9 @@ AST_BEGIN_SUBCLASSES(Expression)       // old AST: Expr
   AST_BEGIN_SUBCLASSES(Decl)           // old AST: Symbol or DefExpr
     AST_NODE(MultiDecl)
     AST_NODE(TupleDecl)
+    AST_NODE(ForwardingDecl)
 
     AST_BEGIN_SUBCLASSES(NamedDecl)
-      //AST_NODE(ForwardingDecl)
       //AST_NODE(TypeDecl)
```

Comment:
Since we already declare TypeDecl below, this comment is no longer useful. Would you mind removing it in this PR since you are adjusting the line next to it? thanks.